uint16 order_id
lego_brick[] bricks
